Static/AnalysisStructured.hs

```diff
@@ -680,7 +680,7 @@ gsigUnionMaybe lg both mn gsig = case mn of
 
 anaExtraction :: LogicGraph -> LibEnv -> DGraph -> NodeSig -> NodeName -> Range ->
               EXTRACTION -> Result (NodeSig, DGraph)
-anaExtraction lg libEnv dg nsig name rg _extr = do
+anaExtraction lg libEnv dg nsig name rg (ExtractOrRemove _ iris _) = do
```

Comment:
if the first argument of `ExtractOrRemove` is `False`, throw an error "remove not implemented yet". This is better than silently ignoring the difference between `extract` and `remove`.